bin/map2reg

```diff
@@ -136,20 +140,12 @@ def main():
     outreg = parallel_map(get_region, uniq_vals,
                           numcores=int(pars["nproc"]))
 
-    # Original version adding CXCRegion objects, but it ended up
-    # consuming a huge amount of memory and was slow.  Writing to
-    # an ASCII region file and then reading back in as a single
-    # CXCRegion is much faster
-
-    # Combine into single region file
-    tmpfile = NamedTemporaryFile(dir=os.environ["ASCDS_WORK_PATH"],
-                                 suffix=".reg", delete=True)
-    with open(tmpfile.name, "w") as outfile:
-        for myreg in outreg:
-            outfile.write(myreg+"\n")
+    # Unfortunately writing all out to an ascii file sometimes fails
```

Comment:
I assume there's no easy (or maybe any) way to tell when the dump to an ASCII file approach will work?

Reply:
There's probably a per-line string length defined somewhere in the DM (it parses the `region()` syntax and sends lines to region lib).  Probably some variation of n*1024 +/- m depending on how new-line/string terminator are handled.